components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/Error.java

```diff
@@ -25,9 +25,11 @@
     INVALID_OAUTH_CLIENT("60002"),
     AUTHENTICATED_USER_NOT_FOUND("60003"),
     FORBIDDEN("60004"),
-    GONE("60005"),
     INVALID_UPDATE("60005"),
     INVALID_DELETE("60006"),
+    GONE("60007"),
```

Comment:
This was a duplicate error code scenario, decided to fix the one with the least usages